libyul/backends/evm/EVMDialect.cpp

```diff
@@ -120,6 +120,13 @@ std::set<YulString> createReservedIdentifiers(langutil::EVMVersion _evmVersion)
 		return _instr == evmasm::Instruction::BASEFEE && _evmVersion < langutil::EVMVersion::london();
 	};
 
+	// TODO remove this in 0.9.0. We allow creating functions or identifiers in Yul with the name
+	// blobbasefee for VMs before cancun.
+	auto blobBaseFeeException = [&](evmasm::Instruction _instr) -> bool
+	{
+		return _instr == evmasm::Instruction::BLOBBASEFEE && _evmVersion < langutil::EVMVersion::cancun();
+	};
```

Comment:
Maybe it would be nice a little refactor, not in this PR though. Because we will now add more three exception annonymous functions here for the cancun support. So we could at least have a proper function to create such exceptions, since they are basically the same. Only `prevrandao` has a different logic.

Reply:
I thought about doing exactly this, but didn't wanna introduce yet another thing to review, especially seeing as we'll be releasing `0.9.0` sometime this year, and this will get removed anyway. As in - we could certainly refactor this to look nicer, but I don't think it's really worthwhile since it's gonna get tossed anyway.